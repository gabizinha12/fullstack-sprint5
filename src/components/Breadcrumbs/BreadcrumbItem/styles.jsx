import styled from "styled-components";

export const ListItem = styled.li`
  font-size: 0.8125rem;
  font-weight: 600;
  color: #585858;
  margin-right: 10px;
  @media (max-width: 1200px) {
    display: none;
  }
  &:not(:last-child):hover {
    text-decoration: underline;
  }
`;

export const ItemLink = styled.a`
  font-size: 0.8125rem;
  font-weight: 600;
  color: ${({ theme }) => theme.text};
  margin-right: 10px;
`;
