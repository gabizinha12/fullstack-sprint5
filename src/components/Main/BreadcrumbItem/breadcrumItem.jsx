import React from "react";

function BreadcrumbItem({ value }) {
  return (
    <li class="breadcrumbs__item breadcrumbs__item--active">
      <span class="breadcrumbs__link">{value}</span>
    </li>
  );
}

export default BreadcrumbItem;
