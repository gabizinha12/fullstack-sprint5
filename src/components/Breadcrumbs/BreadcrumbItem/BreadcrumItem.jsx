import React from "react";
import { ListItem, ItemLink } from "./styles";
function BreadcrumbItem({ value }) {
  return (
    <ListItem>
      <ItemLink>{value}</ItemLink>
    </ListItem>
  );
}

export default BreadcrumbItem;
