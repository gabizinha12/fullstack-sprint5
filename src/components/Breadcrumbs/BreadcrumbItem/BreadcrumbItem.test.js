import { render, screen } from "@testing-library/react";
import { BreadcrumbItem } from "./BreadcrumbItem";

describe("Teste de snapshot BreadcrumbItem", () => {
  it("should render BreadcrumbItem", () => {
    const { container } = render(<BreadcrumbItem value="aaa" />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
