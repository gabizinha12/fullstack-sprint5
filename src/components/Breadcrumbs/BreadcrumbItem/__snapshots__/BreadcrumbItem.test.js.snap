// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste de snapshot BreadcrumbItem should render BreadcrumbItem 1`] = `
<li
  class="sc-bdnxRM iHSEBR"
>
  <a
    class="sc-gtsrHT hToJZB"
  >
    aaa
  </a>
</li>
`;
